deep_quoridor/src/agents/alphazero/alphazero.py

```diff
@@ -242,6 +232,7 @@ def __init__(
                 self.initial_temperature = 0
 
         self.replay_buffer = deque(maxlen=params.replay_buffer_size)
+        self.replay_buffers_in_progress = []
```

Comment:
Now we store the replay buffer(s) in progress (i.e. the current game or games) here, and when the game is done we set the value for each entry and store it in the replay_buffer